sig-operations/charter.md

```diff
@@ -0,0 +1,38 @@
+# SIG Operations
+
+## Scope
+
+The goal of this SIG is to enable management and operations for hardware and platform tooling within the Operate First Community Cloud (op1st)
+
+### In scope
+
+* Define what is part and not part of the Operate First Cloud platform offering
+* Communicate a platform roadmap
+* Day to day operations of platform
+* Operating and ensuring stability of:
+    * Platform management tools
+    * Services provided to workloads
+    * Platform component upgrades
```

Comment:
`platform component upgrades` == upgrading cluster versions? 
`services provided to workloads` - such as? i'm trying to wrap my head around it is all
`platform management tools` == GitOps & Automation tools?

Reply:
This was added by @durandom  so he can clarify but the way I understand it: 

> platform component upgrades == upgrading cluster versions? 

Yeah I think for the most part it's going to be clusters, though we're opting for broader terminology so as not to be limiting to only ocp clusters in the future.

>  services provided to workloads - such as? i'm trying to wrap my head around it is all 

This is probably referring to services like ODH, that are not directly related to operating the cloud and instead offer user's services like JupyterHub, Superset, etc. -- These are managed by the operations team. Though I think it we ought to distinguish between odh-operations (hopefully done by the ODH team) and cluster management, but maybe we can do that via  sub projects?

>  platform management tools == GitOps & Automation tools?

Yep, services like ArgoCD, ACM, Prow, etc.

Reply:
service = OpenDataHub? AIOps?

Reply:
thanks for the info! i'm getting a clearer idea now.

Reply:
hrmm maybe "Services provided to workloads" is a bit confusing, I'm understanding this as: 

"Management of regular services not directly tied to cluster management, for example ODH, Strimzi, Grafana, Openshift Pipelines, etc."

Reply:
@durandom can clarify

Reply:
@HumairAK right, that's what I'm wondering
"Services made available to workloads"? 
"Platform services that are made available to workloads"?

Reply:
Platform components == CNV
Services provided == Observatorium, Camel-K
Platform management tools == ACM

I would put ODH into the user workload bucket. But since we say on bullet point 1

> Define what is part and not part of the Operate First Cloud platform offering

we could shrink it to 

```
* Communicate a platform roadmap
* Day to day operations of the platform
* Ensuring the stability of the platform
* Upgrading the platform
```

Reply:
sounds good to me, done.